src/Phinx/Config/Config.php

```diff
@@ -313,6 +313,18 @@ public function getTemplateClass()
         return $this->values['templates']['class'];
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getDataDomain()
+    {
+        if (!isset($this->values['data_domain'])) {
+            return false;
```

Comment:
if this method returns always array, why not returning `[]` here?
Cleaner return codes.

Reply:
I was following the pattern from the methods above (`getTemplateFile`, `getTemplateClass`), but thinking again you're right. I'll make the return value always an array. Thank you!